pkg/function/scale_workload.go

```diff
@@ -92,14 +93,16 @@ func (*scaleWorkloadFunc) Arguments() []string {
 		ScaleWorkloadNamespaceArg,
 		ScaleWorkloadNameArg,
 		ScaleWorkloadKindArg,
+		ScaleWorkloadWaitArg,
 	}
 }
 
-func getArgs(tp param.TemplateParams, args map[string]interface{}) (namespace, kind, name string, replicas int32, err error) {
+func getArgs(tp param.TemplateParams, args map[string]interface{}) (namespace, kind, name string, replicas int32, waitForReady bool, err error) {
 	var rep interface{}
+	waitForReady = true
 	err = Arg(args, ScaleWorkloadReplicas, &rep)
 	if err != nil {
-		return namespace, kind, name, replicas, err
+		return namespace, kind, name, replicas, waitForReady, err
```

Comment:
I think we can just do a simple `return` without the trailing arguments. Same with line 148, 152, 156, 160 and 162. Sorry, should have pointed that out earlier.